web_services/foxx/brightid/db.js

```diff
@@ -319,168 +166,145 @@ function invite(inviter, invitee, groupId, data, timestamp) {
 }
 
 function dismiss(dismisser, dismissee, groupId, timestamp) {
```

Comment:
Need to check if the member being removed is the head of a family group, and if so, remove the `head` attribute from the group.